sphinx-search-as-you-type.rst

```diff
@@ -0,0 +1,113 @@
+.. post:: July 15, 2019
+   :tags: gsoc, search, feature, in-doc-search
+   :author: Vaibhav
+   :location: LKO
+
+Search Update
```

Comment:
This could definitely use a better title. We should mention what we updated, instead of being vague.

Reply:
I am not sure about what would be a better title.
What about: `Search Update: Section Linking & Search As You Type Feature`

Reply:
I think search as you type is the biggest feature and it should definitely be in the title. I wouldn't be opposed to "Adding Search as You Type to Read the Docs" or something like that.

Reply:
Yea, I'd like to make this title better before merging.

Reply:
I don't have very strong opinion but I do think that `Adding Search as You Type to Read the Docs` focuses only on **one** feature?

Reply:
"Search Update" means nothing, we need to tell people what was updated. I don't feel strongly what we focus on, but it needs more.

Reply:
I am not in favor of `Search Update` also.
How about -- `Adding Search as You Type to Read the Docs and much more`?
Or we can just go with `Adding Search as You Type to Read the Docs`.

Reply:
`GSOC 2019: Improved Search Results and Search As You Type`?

Reply:
Loved that one. :heart:

Reply:
Updated